packages/hooks/src/useGetStatus.ts

```diff
@@ -0,0 +1,27 @@
+import { useStore } from '@deriv/stores';
+import useGetMFAccountStatus from './useGetMFAccountStatus';
+import useIsSelectedMT5AccountCreated from './useIsSelectedMT5AccountCreated';
+import { CFD_PLATFORMS } from '@deriv/shared';
+
+const useGetStatus = () => {
```

Comment:
Would be nice to add a documentation to all the hooks added, to ensure the functionality of the hooks are more clear.